glide.yaml

```diff
@@ -67,3 +67,7 @@ import:
   version: 335da9dda11408a34b64344f82e9c03779b71673
   repo: https://github.com/openshift/glog.git
   vcs: git
+- package: github.com/briandowns/spinner
+  version: 48dbb65d7bd5c74ab50d53d04c949f20e3d14944
```

Comment:
@budhrg Maintainer just released https://github.com/briandowns/spinner/releases 1.0, lets use it instead sha hash